Apps/APN/package.json

```diff
@@ -8,15 +8,16 @@
     "bundle:ios": "react-native bundle --entry-file index.js --platform ios --dev false --bundle-output ios/main.jsbundle --assets-dest ios",
     "clean-install": "./scripts/clean-install-dependencies.sh",
     "clean-install-android": "./scripts/clean-run-android.sh",
-    "clean-install-ios": "./scripts/clean-run-ios.sh"
+    "clean-install-ios": "./scripts/clean-run-ios.sh",
+    "dev:update": "../../scripts/dev:update.sh && pod update --repo-update --project-directory=ios"
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.11",
     "@react-native-clipboard/clipboard": "^1.11.2",
     "@react-native-community/push-notification-ios": "^1.10.1",
     "@react-navigation/native": "^6.0.10",
     "@react-navigation/native-stack": "^6.6.2",
-    "customerio-reactnative": "../..",
+    "customerio-reactnative": "file:../../customerio-reactnative-1690212805.tgz",
```

Comment:
From script below, `1690212805` looks like the timestamp which changes every time. So I don't think we should be changing dependency here, it may fail on customer's machines and may be in CI builds as well?

Also, we'll then probably have to revert this every time before pushing? Considering this saves a lot of our testing time while development, I'm happy with it. But if we can find a way that can prevent this being pushed to git, that will be awesome.

Reply:
That's a good idea. I didn't consider the idea that if we commit the file like it is now, it would not allow others to compile the sample app. 

One idea to resolve this is to use `npm` instead of `yarn` for the sample apps (we can continue to use yarn for the SDK) because `npm` might not have the same cache issue that `yarn` has which is why we have more complex configuration here. We could then change the `package.json` file to use a static file path: `"customerio-reactnative": "file:../../customerio-reactnative.tgz"` instead of it being dynamic like it is now. 

I know `yarn` is very popular in RN so I am not sure if `npm` will work as well. But, it is worth a try. 

Note: The use of using `.tgz` over `../../` is (1) it speeds up `yarn install` for the sample apps and (2) it's installing the SDK the same way that customers installing our module from npmjs.com does. If we use `../../`, we run the risk of passing QA but missing files when we deploy to npm. If we use the `.tgz` file to install from, we are QA testing against a set of files that will be pushed to production. 

I'll try to get this working.

Reply:
Yeah, I was talking about customers trying to build the sample app and not the SDK, sorry for the confusion.

I like the idea of having static file path, if it works well, I think we can use `npm` instead of `yarn` for sample apps 👍🏻

Reply:
I have moved over to npm for the sample apps and it has resolved this problem. Our sample apps can now use a hard-coded value in package.json